libs/shapes/Triangle.cpp

```diff
@@ -0,0 +1,86 @@
+#include "Triangle.h"
+
+namespace soo {
+
+TriangleVertices::TriangleVertices(const ofVec2f& A, const ofVec2f& B,
+                                   const ofVec2f& C) {
+  // Compute the length of the edges
+  float edge_a_length = B.distance(C);
+  float edge_b_length = A.distance(C);
+  float edge_c_length = B.distance(A);
+
+  // Check if the given vertices define a triangle
+  if (!TriangleInequalityTheorem(edge_c_length, edge_b_length, edge_a_length)) {
```

Comment:
for style and reading purposes, change the order of the input lengths to follow a, b, c